### DISCUSSION TITLE
how many extra packages are installed compared to standard install?

### DISCUSSION THREAD

**Discussion by gvian** [archlinux/archinstall#159] [Q&A] [answered]
frankly, i was intrigued, frustated and finally installed arch on laptop. learnt a lot. frequently long time arch users say gui installers install lot of unnecessary packages.

do we have statistics about how many extra packages compared to standard install for a particular or test desktop or laptop?

**Comment by Torxed** [comment, marked as answer]
Yepp, that's coming in `v2.2.0`. But the packages are these:
 * https://github.com/archlinux/archinstall/blob/torxed-v2.2.0/archinstall/lib/installer.py#L14

And whatever packages are needed for the file system chose, for `btrfs` that would be `btrfs-progs` for instance.
And if you choose a profile, say `awesome` the additional ones would be:
 * https://github.com/archlinux/archinstall/blob/torxed-v2.2.0/profiles/awesome.py#L5
 
But in general I try to keep it as close to *"absolutely the only ones that's needed for a functioning system"*, with the exception where it's very optional things such as profiles that are in the best interest of user friendliness. Those tend to hog a log of packages in general, like gnome and all it's "basic needs" are like ~8G or something :)

Tl;dr: `base`, `base-devel`, `linux`, `linux-firmware`, `efibootmgr` + file system packages

**Reply by asbachb** [reply]
@Torxed Is there a reason why `base-devel` is installed by default? Is there a way to prevent that via `user_configuration.json`?

**Reply by Torxed** [reply]
We could remove it I suppose. I don't recall a specific reason for it being there other than this was one of my default packages back in ~2014. It does pull in `sudo` and and `which`, as well as being an optional pacman dependency.

But in reality, submitting a PR with a suggestion to remove it is welcome and we'll see the community feedback (if any) :)
